core/includes/install.core.inc

```diff
@@ -755,14 +785,8 @@ function install_display_output($output, $install_state) {
  * @param $install_state
  *   An array of information about the current installation state.
  *
- * @return string|NULL
+ * @return array
```

Comment:
The return type can't actually be `array` here, as that's not what install_display_requirements() returns (string|NULL).

Reply:
Fixed